pipelines/vars/foreman_1.20.yml

```diff
@@ -0,0 +1,4 @@
+forklift_name: "pipeline-foreman-1.20-{{ foreman_os }}"
+forklift_boxes:
+  pipeline-foreman-1.20-centos7: "{{ server_box }}"
```

Comment:
This probably needs to copy the pattern from nightly to include the `foreman_os`. Same in the 1.21 file.